devel/isl-devel/Portfile

```diff
@@ -1,15 +1,112 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           obsolete 1.0
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           muniversal 1.1
 
 name                isl-devel
-replaced_by         isl
+conflicts           isl
+set my_name         isl
 
-set gitSHA1         f8b5d3df1e4dd2c4a300d25954d344b757347f2c
-version             0.21-20190404-[string range $gitSHA1 0 6]
-revision            1
+# for library name changes, cross-compilers do *not* show up in traditional dependency searches
+# (e.g. port file all | sort -u | xargs grep -E ':isl( |$)' | cut -d / -f 13 | sort -u)
+# see https://lists.macports.org/pipermail/macports-dev/2019-May/040678.html
+epoch               4
+version             0.25
+revision            0
 
 categories          devel math
+license             MIT
+maintainers         nomaintainer
 
-# Note: Port obsoleted 2021-12-20
+description         Integer Set Library
+long_description    isl is a library for manipulating sets and \
+                    relations of integer points bounded by linear \
+                    constraints. Supported operations on sets include \
+                    intersection, union, set difference, emptiness \
+                    check, convex hull, (integer) affine hull, integer \
+                    projection, computing the lexicographic minimum \
+                    using parametric integer programming, coalescing \
+                    and parametric vertex enumeration. It also \
+                    includes an ILP solver based on generalized basis \
+                    reduction, transitive closures on maps (which may \
+                    encode infinite graphs), dependence analysis and \
+                    bounds on piecewise step-polynomials.
+homepage            https://libisl.sourceforge.io
+
+depends_lib         port:gmp
+
+master_sites        sourceforge:libisl
+distname            ${my_name}-${version}
+use_bzip2           yes
+
+checksums           rmd160  73b9005b8e7fe52eb9500b98ba65450bdfb9f7fb \
+                    sha256  4305c54d4eebc4bf3ce365af85f04984ef5aa97a52e01128445e26da5b1f467a \
+                    size    2304378
+
+if {${subport} eq ${name}} {
+    platform darwin {
```

Comment:
does this need to be `platform darwin`? if not remove this clause

Reply:
Yes, this is needed. All stuff inside the clause is Mac-specific.

Reply:
Gotcha. OK makes sense :+1: